src/packageurl/contrib/purl2url.py

```diff
@@ -470,3 +540,24 @@ def get_repo_download_url(purl):
     return get_repo_download_url_by_package_type(
         type=type, namespace=namespace, name=name, version=version
     )
+
+
+# TODO: https://github.com/package-url/packageurl-python/issues/196
+def escape_path(path: str) -> str:
```

Comment:
Can you rename this?
```suggestion
def escape_golang_path(path: str) -> str:
```